### PR TITLE
Update REQ-DCW-06 DICOMWeb STOW-RS

### PR DESCRIPTION
**Pull request by mocsharp**
### Description

Updates the requirements for REQ-DCW-06 DICOMWeb STOW-RS.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] All tests passed locally.
- [ ] [Documentation comments](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments) included/updated.
- [ ] User guide updated.
- [ ] I have updated the changelog

### CODE REVIEW

File: guidelines/mig-drd.md [hunk 1]
```diff
@@ -277,11 +277,16 @@ TBD
 ### [REQ-DCW-06] MIG SHALL be able to allow users to upload DICOM objects via DICOMweb STOW-RS
```
Comment (rahul-imaging) [inline guidelines/mig-drd.md:277]: Will you add a new requirement for allowing submission of a job to MONAI Deploy via STOW-RS using a REST call ?

**Comment by rahul-imaging** [comment]
The changes look good to me

File: guidelines/mig-drd.md (hunk 1 above)
Comment (dbericat) [reply guidelines/mig-drd.md:277]: I was assuming it was over REST. If not, what other mechanism does STOW-RS use?

File: guidelines/mig-drd.md [hunk 2]
```diff
@@ -420,6 +425,14 @@ MONAI Deploy Informatics Gateway R1
 Since MIG is the integration point with the medical systems for MONAI Deploy, it shall notify other subsystems when data is received and ready.
 
 
+#### Verification Strategy
+
+Send a dataset to MIG and expect MIG to emit a message to notify other subsystems of the location of the data.
+
+#### Target Release
+
+MONAI Deploy Informatics Gateway R1
```
Comment (dbericat) [inline guidelines/mig-drd.md:434]: Why is R1 here and 0.2.0 above? Shouldn't it be the same?

**Review by dbericat — CHANGES_REQUESTED** [review]
Added a couple comments. Thanks!

File: guidelines/mig-drd.md (hunk 1 above)
Comment (mocsharp) [reply guidelines/mig-drd.md:277]: `REQ-FNC-02` covers all methods of triggering workflows, e.g. C-STORE, ACR, and in this case, STOW-RS>

File: guidelines/mig-drd.md (hunk 2 above)
Comment (mocsharp) [reply guidelines/mig-drd.md:434]: This is for a `REQ-FNC-02` where I forgot to include the target release and test strategy.

**Review by dbericat — APPROVED** [review]
LGTM

**Pull request merged**